src/apps/olympus/ethereum/olympus.balance-fetcher.ts

```diff
@@ -61,18 +63,44 @@ export class EthereumOlympusBalanceFetcher implements BalanceFetcher {
     });
   }
 
+  private async getV2Bonds(address: string) {
+    const network = Network.ETHEREUM_MAINNET;
+    return this.contractPositionV2BalanceHelper.getBalances<OlympusV2BondDepository>({
+      network,
+      groupId: OLYMPUS_DEFINITION.groups.bondV2.id,
+      appId: OLYMPUS_DEFINITION.id,
+      address,
+      resolveDepositoryContract: ({ depositoryAddress: address }) => {
+        return this.contractFactory.olympusV2BondDepository({ address, network });
+      },
+      resolveClaimablePayout: ({ multicall, contract, address }) =>
+        multicall
+          .wrap(contract)
+          .indexesFor(address)
+          .then(
+            async indexes =>
+              await Promise.all(indexes.map(index => multicall.wrap(contract).pendingFor(address, index))),
+          ),
+    });
+  }
+
   async getBalances(address: string) {
-    const [stakedAssets, bonds] = await Promise.all([this.getTokenBalances(address), this.getBonds(address)]);
+    const [stakedAssets, bonds, bondsV2] = await Promise.all([
+      this.getTokenBalances(address),
+      this.getBonds(address),
```

Comment:
do you want to remove V1 Bonds? They shouldn't exist anymore.

Reply:
I went back and forth on this, but yeah its the right call since we also pulled it from our frontend. I just refactored v1 bonds out. should make reviewing this PR a little easier as well.